README.md

```diff
@@ -127,4 +127,4 @@ curl -X POST http://localhost:8000/messages \
 This example demonstrates sending a routing message to the mediator, which it will forward to the intended recipient.
```

Comment:
Where is the example

Reply:
It's not necessary since you asked for it to be removed and placed in another file.